### ISSUE TITLE
Feature Request: allow / disallow extensions in container

### ISSUE THREAD

**Issue by Conan1989** [mozilla/multi-account-containers#1695]
Hi,
The ability to control what extensions can run in which containers would be very handy.

eg.
Allow uBlock Origin to run in the default container
Disallow all extensions in banking container

**Comment by v-zhuravlev** [comment]
Yes, For example,I have some shopping advisor extensions that I want to keep only in shopping container.

**Comment by rr0ss0rr** [comment]
+1 Great Idea

**Comment by Solid-Ice8** [comment]
'Default' means no container, you may want to add Temporary Container (and also enable 'Automatic Mode' under "Options".

And this sounds like you want to use Wildcards (*) for sub-domains.

Example:

You assign "amazon.com" to Shopping.

Amazon opens sites like:

abc.amazon.com --> You manually add to Shopping container
acb.amazon.com --> You manually add to Shopping container
bac.amazon.com --> You manually add to Shopping container
cba.amazon.com
cab.amazon.com 

And the list goes on..... You can add a wildcard  (*) =  *.amazon.com <== All of these will open in Shopping automatically without manually adding them.

Is this what you mean?

If it is, you can use Simple Tabs Group. After creating a group, go to "Edit Group Setting" and this screen will be most helpful screen ever. You can enable wildcard use (*.amazon.com) near the bottom.

**Comment by Conan1989** [comment]
Hi Solid-Ice8,
Sorry, was thinking of something a little different.
Idea is to run browser extensions inside an 'Account Container', instead of globally. A bit like how you can chose not to run an extension in 'Private Window' mode.

For a use case like uBlock Origin, great for general web browsing. But you probably don't want that running in the tab used for accessing your bank, Azure, AWS, or anything else privileged.

Running two instances of Firefox (Firefox normal, Firefox dev editon) achieves the segregation, but not ideal

**Comment by kyounger** [comment]
Yes! 

This, coupled with Temporary Containers, would be a huge boon for security and privacy. It would allow locking down really hard the default and temporary containers, and then selectively disabling extensions for certain containers one might deem more trustworthy. 

Additionally, you could disable certain extensions in a temporary container that would make a particular site function (and maybe start tracking you; whatever it's in a temp container!), and that brief potential security degradation is short-lived and scoped to the life/reach of the temporary container (versus many extensions allowing you to enable/disable per domain, which you might forget about re-enabling).

**Comment by dannycolin** [comment]
Duplicate of #1167
